readthedocs/templates/core/project_detail_right.html

```diff
@@ -95,13 +95,8 @@ <h3>{% trans "Project Privacy Level" %}</h3>
 {% endblock %}
 
 {% block short-urls %}
-  <h3>{% trans "Short URLs" %}</h3>
-  <p>
-    <a href="http://{{ project.slug}}.{{ PUBLIC_DOMAIN }}">{{ project.slug }}.{{ PUBLIC_DOMAIN }}</a><br>
-    {% if PUBLIC_DOMAIN == "readthedocs.io" %}
-    <a href="http://{{ project.slug}}.rtfd.io">{{ project.slug }}.rtfd.io</a>
-    {% endif %}
-  </p>
+  <h3>{% trans "URLs" %}</h3>
+  <p><a href="{{ project.get_docs_url }}">{{ project.slug }}</a></p>
```

Comment:
@francescozaia what do we want as label here?  the slug, the complete url (which is fairly long) or the  title ?

Reply:
I would use the full URL, so that the user can have a reference to copy+paste it. Said that, IMHO that part can be removed altogether, as the link is the same as shown on the top of the page, which is quite prominent.

Reply:
@protoroto ok, let's completely remove this part of the template